### ISSUE TITLE
Video feature extraction

### ISSUE THREAD

**Issue by leomqyu** [YuanGongND/cav-mae#33]
Hi Yuan thank you for your valuable source code! I really like your `src/extract_audio_representation.py` code on how to extract features from audios as it is clear and self contained.

Also, I am trying to use the model to extract features from video data, and I am trying to build the code depending on `extract_audio_representation.py`. I understand at line 40:
`audio_output, _ = audio_model.module.forward_feat(audio_input, video_input)` 
the second output should be the video_output and could be potentially used as the video dataset, but I am not sure if this is the only modification we need.

For example, in line 67:
`val_loader = torch.utils.data.DataLoader(dataloader.AudiosetDataset(args.data_val, label_csv=args.label_csv, audio_conf=val_audio_conf), batch_size=batch_size, shuffle=False, num_workers=32, pin_memory=True)`

the dataset we use is dataloader.AudiosetDataset which is also in the source code, but we don't have something called a dataloader.VideoDataset. I don't know if it is the case that when we are encoding video features, we could also use dataloader.AudiosetDataset.

Thank you for your explanation in advance! And I would be appreciated if you could also release the `extract_video_representation.py` if available!
